build.sh

```diff
@@ -10,61 +10,19 @@ SCRIPT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)
 ###########################################################################
 
 BUILD_PROJECT_FILE="$SCRIPT_DIR/build/_build.csproj"
-TEMP_DIRECTORY="$SCRIPT_DIR/.nuke/temp"
+TEMP_DIRECTORY="$SCRIPT_DIR//.nuke/temp"
 
-DOTNET_GLOBAL_FILE="$SCRIPT_DIR/global.json"
+DOTNET_GLOBAL_FILE="$SCRIPT_DIR//global.json"
 DOTNET_INSTALL_URL="https://dot.net/v1/dotnet-install.sh"
 DOTNET_CHANNEL="STS"
 
 export DOTNET_CLI_TELEMETRY_OPTOUT=1
 export DOTNET_NOLOGO=1
-export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
-export DOTNET_MULTILEVEL_LOOKUP=0
-
 
 ###########################################################################
 # EXECUTION
 ###########################################################################
 
-#### START CUSTOM CODE ####
```

Comment:
When matt [removed support for net6 and 7](https://github.com/OctopusDeploy/OctoVersion/pull/232) we didn't need to install those SDK's anymore. 

Also now that we're building with github actions, using the GHA workflow to install the extra .NET SDK's is much cleaner than modifying the nuke bootstrapper, so this restores it to the state that Nuke expects.